docs/try-viam/rover-resources/rover-tutorial.md

```diff
@@ -158,7 +158,7 @@ This is the recommended order to assemble your rover:
 
 ### Install Raspberry Pi OS
 
-Install a 64-bit Raspberry Pi OS with the [Raspberry Pi imager](https://www.raspberrypi.com/software/) and put it in your Pi’s microSD card slot.
+Install a 64-bit Raspberry Pi OS with the [Raspberry Pi imager](https://www.raspberrypi.com/software/) and put it in your Pi’s microSD card slot. Don't forget to enabling i2c using raspi-config command so that you can use the accelerometer on the rover.
 For more detailed instructions, check out our Raspberry Pi [installation guide](/installation/prepare/rpi-setup/).
```

Comment:
Since we are sending users to a different resource here, let's reshuffle how we introduce this so that users know to be looking for a particular step on the target resource:

```
Next, install a 64-bit Raspberry Pi OS onto your Pi following our [Raspberry Pi installation guide](/installation/prepare/rpi-setup/).
Be sure to follow all steps as listed, including enabling `i2c` using the `raspi-config` command, as described in [Enable communication protocols](/installation/prepare/rpi-setup/#enable-communication-protocols).
```

Reply:
though it now makes the step right after it useless: "For more detailed instructions, check out our Raspberry Pi [installation guide](/installation/prepare/rpi-setup/)." i think what people don't understand is what the i2c enabling does and why/when do they need it, so stating that it allows them to use the accelerometer is a good detail to add?

Reply:
Ooops, yes I recommend deleting that line also. Sorry I thought I used the suggestion edit mode and deleted it. You're right -- good eye!

Reply:
How about the following to also fit in that I2C is for enabling use of the accelerometer:

```
Next, install a 64-bit Raspberry Pi OS onto your Pi following our [Raspberry Pi installation guide](/installation/prepare/rpi-setup/).
Follow all steps as listed, including the final step, [Enable communication protocols](/installation/prepare/rpi-setup/#enable-communication-protocols), which is required to enable the accelerometer on your rover.
```

(and then deleting that final dupe line also)

Reply:
added!